deploy/determined_deploy/aws/deployment_types/efs.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+
+import boto3
+import pkg_resources
+
+from determined_deploy.aws import aws, constants
+from determined_deploy.aws.deployment_types import base
+
+
+class EFS(base.DeterminedDeployment):
```

Comment:
It seems like this and the `FSx` class are identical, could we avoid duplicating this code?